tests/plugins/neg/divideZero-research/plugin_1.scala

```diff
@@ -10,13 +10,13 @@ import Symbols.Symbol
 import Constants.Constant
 import StdNames._
 
-class DivideZero extends MiniPhase with ResearchPlugin {
+class DivideZero extends PluginPhase with ResearchPlugin {
```

Comment:
Since the CI only run the sbt scripted tests on nightlies, can you confirm that you ran this test locally

Reply:
This test is a neg test, not a scripted test. I just checked it works locally. I confused my self, for research plugins, we don't need the phases to be `PluginPhase`.